src/coprocessor/dag/aggr_fn/parser.rs

```diff
@@ -41,11 +42,15 @@ pub trait AggrDefinitionParser {
 
 #[inline]
 fn map_pb_sig_to_aggr_func_parser(value: ExprType) -> Result<Box<dyn AggrDefinitionParser>> {
+    use std::marker::PhantomData;
     match value {
         ExprType::Count => Ok(Box::new(super::impl_count::AggrFnDefinitionParserCount)),
         ExprType::Sum => Ok(Box::new(super::impl_sum::AggrFnDefinitionParserSum)),
         ExprType::Avg => Ok(Box::new(super::impl_avg::AggrFnDefinitionParserAvg)),
         ExprType::First => Ok(Box::new(super::impl_first::AggrFnDefinitionParserFirst)),
+        ExprType::Agg_BitAnd => Ok(Box::new(AggrFnDefinitionParserBitOp::<BitAnd>(PhantomData))),
```

Comment:
It's not wise to expose `PhantomData` to the caller. Implement `AggrFnDefinitionParserBitOp::<BitAnd>::new()` instead.

Reply:
addressed